ddtrace/tracer/textmap.go

```diff
@@ -322,6 +359,24 @@ func (p *propagator) extractTextMap(reader TextMapReader) (ddtrace.SpanContext,
 	return &ctx, nil
 }
 
+func extractPropagatingTags(ctx *spanContext, v string, err error) {
```

Comment:
Please document this function. What does it do? What does it extract? It has no return value.